src/main/resources/main.css

```diff
@@ -489,181 +493,26 @@ VBox.spaced, HBox.spaced {
 .labeled-link > .label {
     -fx-font-weight: bold;
 }
-
-.button-qrcode {
-    -fx-background-repeat: no-repeat;
-    -fx-background-image: url('/img/settings.png');
-    -fx-background-position: left;
-    -fx-pref-height: 35px;
-    -fx-pref-width: 40px;
-    -fx-border-style: none;
-    -fx-border-color: none;
-}
-
-.button-qrcode:hover {
-    -fx-background-color: transparent;
-    -fx-background-insets: 0px;
-    -fx-border-style: none;
-    -fx-border-color: none;
-    -fx-text-fill: #ff690f;
-}
-
-.button-qrcode:pressed {
-    -fx-border-color: none;
-    -fx-text-fill: none;
-    -fx-border-style: none;
-}
-
-.pane-popup {
-    -fx-background-color: rgba(0,0,0,0.8);
-    -fx-padding: 50px 0px 50px 0px;
-}
-
-.icon-client {
-    -fx-background-repeat: no-repeat;
-    -fx-background-image: url('/img/account.png');
-}
-
-.icon-earth{
-    -fx-background-repeat: no-repeat;
-    -fx-background-image: url('/img/earth.png');
-}
-
-.icon-key {
-    -fx-background-repeat: no-repeat;
-    -fx-background-image: url('/img/key_variant.png');
-    -fx-margin: 20px;
-}
-
-.icon-client, .icon-earth, .icon-key {
-    -fx-background-position: left;
-    -fx-pref-height: 35px;
-    -fx-pref-width: 40px;
-}
-
-.label-icon {
-    -fx-pref-width: 250px;
-    -fx-pref-height: 50px;
-    -fx-padding: 5px 20px 5px 10px;
+.syncItem {
+    -fx-border-color: darkgrey;
+    -fx-border-width: 1 0 0 0;
 }
 
-.label-icon .text {
-    -fx-font-size: 12;
-}
 
-.label-name {
-    -fx-pref-width: 200px;
-    -fx-padding: 5px 0px 5px 10px;
+* {
+    -fx-syncAvatar: 25;
```

Comment:
stick to css best practices / lisp-case: `-fx-sync-avatar`. That name is not very descriptive anyways. It's about size so `-fx-sync-avatar-width` or similar would should do. The name could also describe a font-family or a border size.